Wrappers/Python/cil/optimisation/utilities/callbacks.py

```diff
@@ -134,3 +135,55 @@ class LogfileCallback(TextProgressCallback):
     def __init__(self, log_file, mode='a', **kwargs):
         self.fd = open(log_file, mode=mode)
         super().__init__(file=self.fd, **kwargs)
+        
+class EarlyStoppingObjectiveValue(Callback):
+    '''Callback that stops iterations if the change in the objective value is less than a provided threshold value.
+
+    Parameters
+    ----------
+    threshold: float, default 1e-6 
+
+    Note
+    -----
+    This callback only compares the last two calculated objective values. If `update_objective_interval` is greater than 1, the objective value is not calculated at each iteration (which is the default behaviour), only every `update_objective_interval` iterations.
+    
+        '''
+    def __init__(self, threshold=1e-6):
+        self.threshold=threshold
```

Comment:
VERY strongly disapprove of this. Far superior naming, vis. https://keras.io/api/callbacks/early_stopping/:

- `EarlyStopping`, `EarlyStoppingObjectiveDelta`, `EarlyStoppingObjectiveDelta`, `EarlyStoppingObjectiveChange`
- `delta`, `change`